Cython/Utility/CppSupport.cpp

```diff
@@ -61,7 +61,8 @@ auto __Pyx_pythran_to_python(T &&value) -> decltype(to_python(
 
 ////////////// MoveIfSupported.proto //////////////////
 
-#if __cplusplus >= 201103L
+#if __cplusplus >= 201103L || (defined(_MSC_VER) && _MSC_VER > 1900)
```

Comment:
r-value-reference and `std:move()` are supported starting in VS2010 (`_MSC_VER >= 1600`)

Reply:
Thanks - updated. Practically I think that only affects people still on Python 3.3-3.4 but we may as well get this as right as possible.